src/main/java/Event.java

```diff
@@ -0,0 +1,21 @@
+public class Event extends Task {
+    protected String at;
+
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
+    }
+
+//    public void setAt(String at) {
```

Comment:
Maybe can delete away methods that are not needed or used so that the code looks tidier. But awesome code tidiness!!! - SWJE